pkg/kubelet/cadvisor/util.go

```diff
@@ -29,6 +29,12 @@ import (
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
 )
 
+const (
+	// Please keep this in sync with the one in:
+	// github.com/google/cadvisor/container/crio/client.go
+	CrioSocket = "/var/run/crio/crio.sock"
```

Comment:
This would somehow help solve the building errror. But I still prefer not to have two copies of `CrioSocket`.

Reply:
@dixudx yes, me too. but since we are really short on time for 1.9 .... i logged it just in case you wanted to chase it. i can abandon this tomorrow